database/dbSchoolSuppliesForm.php

```diff
@@ -43,5 +43,39 @@ function isBackToSchoolFormComplete($childID) {
     return $complete;
 }
 
+
+function getSchoolSuppliesSubmissions() {
+    $conn = connect();
+    $query = "SELECT * FROM dbSchoolSuppliesForm JOIN dbChildren USING(id);";
+    $result = mysqli_query($conn, $query);
+    if(mysqli_num_rows($result) > 0){
+        $submissions = mysqli_fetch_all($result, MYSQLI_ASSOC);
+        mysqli_close($conn);
+        return $submissions;
+    }
+    mysqli_close($conn);
+    return [];
+}
+
+function getSchoolSuppliesSubmissionsFromFamily($familyId) {
+    require_once("dbChildren.php");
+    $children = retrieve_children_by_family_id($familyId);
+    if (!$children){
+        return [];
+    }
+    $childrenIds = array_map(function($child) {
+        return $child->getId();
+    }, $children);
+    $joinedIds = join(",",$childrenIds);
+    $conn = connect();
+    $query = "SELECT * FROM dbSchoolSuppliesForm JOIN dbChildren ON dbSchoolSuppliesForm.child_id = dbChildren.id WHERE dbSchoolSuppliesForm.child_id IN ($joinedIds)";
```

Comment:
When I search only by school supplies form, I get the listing of children from dbSchoolSupplies (which is good) but the additional child info for each row (first_name, last_name, dob, gender, medical_notes, and notes) is info that is tied to a completely different child.
<img width="1381" alt="Screenshot 2024-12-03 at 6 51 45 PM" src="https://github.com/user-attachments/assets/3ace5b0e-817c-4a93-846b-93433cda99cf">